homeworks/src/ltretykhlib/module3/Triangle.java

```diff
@@ -0,0 +1,98 @@
+package ltretykhlib.module3;
+
+import static java.lang.Math.*;
+
+/**
+ * Created by Tretykhlib on 23.04.2017.
+ */
+public class Triangle {
+    private double Ax;
+    private double Ay;
+    private double Bx;
+    private double By;
+    private double Cx;
+    private double Cy;
+
```

Comment:
create a separate class Point to encapsulate x,y coordinates and all the logic related to distance calculation between two points